lib/apiservers/engine/backends/container.go

```diff
@@ -1263,7 +1264,7 @@ func (c *ContainerBackend) containerStop(op trace.Operation, name string, second
 	}
 
 	operation := func() error {
-		return c.containerProxy.Stop(op, vc, name, seconds, true)
+		return c.containerProxy.Stop(op, vc, name, seconds, false)
```

Comment:
What is logic from true -> false here?

Reply:
It is using to determine if we need to call unbindContainer.